hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -815,13 +821,14 @@ public void loadAll(Set<K> keys, boolean replaceExistingValues) {
         MapQueryRequest request = new MapQueryRequest(name, predicate, IterationType.ENTRY);
         QueryResultSet result = invoke(request);
```

Comment:
but what if someone add a new item into the `QueryResultSet` after deserialization? We could  have `QueryResultSet` backed by `InflatableSet` but I don't think it's related to this PR.